services/core/shared/core/delegates.js

```diff
@@ -192,7 +192,8 @@ const getNextForgers = async params => {
 	};
 
 	const offset = params.offset || 0;
-	const limit = params.limit || 10;
+	let limit = params.limit || 103;
+	if (sdkVersion < 4) limit = 101;
```

Comment:
If limit specified in params, consider it else the default value.
```suggestion
	if (sdkVersion < 4) limit = params.limit || 101;
```

Reply:
Rolled back to 10.